nixops/network.nix

```diff
@@ -152,5 +152,68 @@
 
       nix.buildCores = 24;
     };
+    packet-spot-eval-4 = { resources, ... }: {
```

Comment:
we should probably make this a more generic at some point so it's easier to scale up/down

Reply:
Done in https://github.com/ofborg/infrastructure/pull/25.